src/components/FilterMenuModal/FilterMenuModal.js

```diff
@@ -155,6 +155,7 @@ const FilterMenuModal = ({
         return (
           <FilterMenuWizardContext.Provider value={{ filterMenuName }}>
             <RoundedIcon
+              id="hub-filter-btn"
```

Comment:
FilterMenuModal is used not only in the Batch run. The specified id is related only to the Batch run